service-deployment/docker_build.py

```diff
@@ -283,6 +312,7 @@ def hadoop_binary_prepare(custom_hadoop_path, hadoop_version):
 def main():
     parser = argparse.ArgumentParser()
     parser.add_argument('-p', '--path', required=True, help="path to cluster configuration file")
+    parser.add_argument('-n', '--name', default='all', help="Build and push an image to the registry")
```

Comment:
>('-n', '--name', d [](start = 23, length = 18)

need to update readme?

Reply:
Expose it in the readme.md in the future.